main.go

```diff
@@ -98,6 +113,21 @@ func main() {
 		os.Exit(0)
 	}
 
+	if *generateSigningKey {
+
+		workingDirectory, err := os.Getwd()
+		if err != nil {
+			fmt.Errorf("Unable to get current working directory: %s", err)
```

Comment:
would it make sense to use `log.Fatal` here just to keep things consistent?

Reply:
Sure. I'm not sure on the difference between `log` and `fmt` - I'll have another look.